pkg/operator/client/deploy.go

```diff
@@ -522,11 +524,27 @@ func (c *Client) installWithHelm(helmDir string, targetNamespace string, kotsCha
 		installDir := filepath.Join(chartsDir, dir.Name)
 		args := []string{"upgrade", "-i", dir.ReleaseName, installDir, "--timeout", "3600s"}
 
+		upgradeNamespace := "default"
```

Comment:
too many variables that refer to a namespace which is pretty confusing. my recommendation is:

- pass the `targetNamespace` to the `getSortedCharts` function and if the `dir.Namespace` is empty, set it to the `targetNamespace` (if it's not `.`.
- change the if statement below to just be `if dir.Namespace != util.AppNamespace()`. e.g.: let's not define 2 more variables that refer to a namespace

Reply:
from what i can see and know, it should never be `.` when you get to this the `installWithHelm` function, so you don't need to check for that